### ISSUE TITLE
Zooming at GF and at all

### ISSUE THREAD

**Issue by Filename99** [ShadowMario/FNF-PsychEngine#8243]
### Describe your problem here.

Hello! I am doing a Tutorial remix and I already did almost everything: made a song and chart, made Vibin' version of GF (as separate character). Also, made a script to delete the Opponent so that BF and GF are the ones who's on the stage. The only thing needed to complete this thing is the zooming script. The problem is that I don't know how to do this. I tried using "defaultCamZoom" (sorry if I misspelled it) for making, you guessed it, default zoom. Something like this:
`function onCreate()`
`	setProperty('defaultCamZoom', 1.1); -- probably 1.1 cuz some shit and stuff`
`end`

And the issue here is that the zoom is not starting at the start of "Three, Two, One, Go!" but if someone starts singing. I did the `setProperty('defaultCamZoom', 1.1)` with the `onStartCountdown()` and `onCountdownTick(counter)` but even then it still triggered when singing occurs. Or even crashed the whole LUA. It would send me errors with no real explanation behind them, like: `<something> is expected near <something>`, when in reality it WAS near something.

And the main problem is zooming at GF. **I CAN'T EVEN THINK OF WHAT TO DO TO CAUSE IT.** I don't know how to do this "Camera loves GF" thing. And this was the main inspiration to make this issue.

So, how to resolve it?

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

All changes are listed in the "Describe the problem" section.

**Comment by frantastic24** [comment]
> made a script to delete the Opponent so that BF and GF are the ones who's on the stage

Instead of making GF the opponent? The engine will automatically handle her positioning and do the zoom tween when it's her turn

> It would send me errors with no real explanation behind them, like: something is expected near something

Issue with your code, not the engine's fault

**Comment by Filename99** [comment]
@frantastic24 
Okay, I rewatched my issue again and pointed out I didn't mention that I'm using a completely separate character for GF. My bad. I can't replace the original GF spritesheet for reasons (that I don't want to list here because I'm too lazy). So I made the script to delete the Opponent 'cuz GFs (on Girlfriend AND Opponent position) were stacking on each other.

About errors, yes, there ARE flaws in my code. But the engine doesn't explain WHAT should I fix. So that's I wrote it there.
